rov/movement/thruster/Force_Limiter.py

```diff
@@ -0,0 +1,117 @@
+from math import floor
+import operator
+
+class ForceLimiter(object):
+
+    # TODO: convertToPowerUnit()
+
+    def __init__(self):
+        self.a6vector = [0 for _ in range(6)]
+        self.an8vector = [0 for _ in range(8)]
+        self.last8vector = [0 for _ in range(8)]
```

Comment:
I don't quite understand the naming here. Can you make more descriptive names? I think they correspond to something like this:
`a6vector = output_force`
`an8vector = thruster_forces`
`last8vector = last_thruster_forces`

Reply:
Agreed

Reply:
update:
a6vector = force_direction
an8vector = thruster_powers
last8vector = prev_thruster_powers